care/users/models.py

```diff
@@ -97,6 +97,7 @@ class LocalBody(models.Model):
     localbody_code = models.CharField(max_length=20, blank=True, null=True)
 
     class Meta:
+        verbose_name_plural = "local bodies"
```

Comment:
@rzr911 where is verbose_name_plural used?

Reply:
So django admin, by default, tries to pluralize the word by adding a 's' at the end of the word.
![image](https://user-images.githubusercontent.com/4143788/121321107-70500080-c92b-11eb-9961-767864ae8ea5.png)

Adding a `verbose_name_plural` value will override it,
![image](https://user-images.githubusercontent.com/4143788/121321361-a7bead00-c92b-11eb-8f70-a6923635a724.png)